string itP_name
---
robocup_msgs/InterestPoint itP